iris/src/iris/web/routers/pipelines.py

```diff
@@ -168,7 +170,8 @@ def run_text_exercise_chat_pipeline_worker(dto, variant_id):
                 break
         else:
             raise ValueError(f"Unknown variant: {variant_id}")
-        pipeline = TextExerciseChatPipeline()
+        is_local = bool(dto.settings and dto.execution.settings.artemis_llm_selection == "LOCAL_AI")
```

Comment:
This will not work, TextExerciseChatPipelineExecutionDTO does not have an execution field. Please test the text exercise chat end to end after fixing this